README.md

```diff
@@ -10,9 +10,23 @@ To view our prototype, clone and enter the following repo:
 git clone git@github.com:project-ivi/ivi.git
 cd ivi
 ```
-Then install and build the sources.
+Install the dependencies:
 ```
 npm install
+```
+During the installation process, some additional steps are necessary to install the semantic-ui module. Navigate the prompts as follows:
+- 'Set-up Semantic UI' > Select `Automatic`
```

Comment:
I thought this would be automated with no CLI input when `autoInstall` is true in `semantic.json`?

Reply:
good catch - i removed this.